src/views/PancakeSquad/components/EventStepsSection/styles.tsx

```diff
@@ -0,0 +1,16 @@
+import { Flex } from '@pancakeswap/uikit'
+import styled from 'styled-components'
+
+export const StyledEventStepsSectionContainer = styled(Flex)<{ $isDark: boolean }>`
+  position: relative;
+  background: ${({ $isDark }) =>
+    $isDark
+      ? 'linear-gradient(180deg, #0B4576 0%, #091115 100%);'
+      : 'linear-gradient(180deg, #6fb6f1 0%, #eaf2f6 100%);'};
```

Comment:
This seems to put `;;` in css. Probably doesn't break anything though.

Reply:
Your probably right, will clean that.